src/openrct2/ride/Ride.cpp

```diff
@@ -397,8 +397,23 @@ money32 Ride::CalculateIncomePerHour() const
 
     if (currentShopItem != SHOP_ITEM_NONE)
     {
-        priceMinusCost += price[1];
-        priceMinusCost -= ShopItems[currentShopItem].Cost;
+        money16 photoProfit = price[1] - ShopItems[currentShopItem].Cost;
+
+        if (lifecycle_flags & RIDE_LIFECYCLE_ON_RIDE_PHOTO)
+        {
+            uint32_t photosSold = no_secondary_items_sold;
+            int32_t rideTicketsSold = total_customers - no_secondary_items_sold;
+
+            // for now, ratio is just using all-time values of tickets sold and photos sold instead of last 5 minutes'
```

Comment:
This comment is stating the obvious, while it doesn't mention _why_ it uses a ratio in the first place.

```suggestion
            // Use the ratio between photo sold and total admissions to approximate the photo income (as not every guest will buy one.
            // TODO: use data from the last 5 minutes instead of all-time values for a more accurate calculation.
```